### ISSUE TITLE
Add basic filter capabilities

### ISSUE THREAD

**Issue by LaurenzV** [linebender/kompari#6]
Firstly, thanks for creating this tool! Will be pretty useful for me in the future. :D

I was wondering, are there any plans to add basic filtering capabilities in the report? The most basic one I can think of is being able to filter out image pairs that match so I can focus on the non-matching ones, but another thing that could be useful is the ability to filter images by name.

**Comment by LaurenzV** [comment]
Ahh, nevermind, I realized that you can ignore matching files from the CLI! I guess the name filtering thing could still be useful though.

Perhaps it would still be useful to have some interactivity on the report page itself, but I understand if this is out of scope for this crate.

**Comment by spirali** [comment]
Yes, you can ignore matching via `--ignore-match`. After your suggestion, I have also created PR #7 that adds `--filter` to CLI that allows to filter by name.

The interactivity is in the scope of this project. My nearest goal is to add some small JS to the report that would add some filtering and image zooming. 

The bigger goal is to have interactive blessing. It is still in discussion how to do it, it would be nice to have a xillem app. I am also thinking about possibility to start a small rest service and make the HTML report more capable in "jupyter-like" style.
